src/js/changeToPressed.js

```diff
@@ -0,0 +1,16 @@
+function changeToPressed(button, forceShiftToggle = false) {
+    const shiftLeft = document.querySelector('.ShiftLeft');
+    const shiftRight = document.querySelector('.ShiftRight');
+
+    if (button.classList.contains('ShiftLeft') || button.classList.contains('ShiftRight') || forceShiftToggle) {
+        shiftLeft.classList.toggle('keyboard__button_activate');
+        shiftRight.classList.toggle('keyboard__button_activate');
+    } else if (button.classList.contains('CapsLock')) {
+        button.classList.toggle('keyboard__button_activate');
+    } else {
+        button.classList.add('active');
+    }
+    return false;
+}
+
+export default changeToPressed;
```

Comment:
I guess, you can combine this function with changeToUnpressed

Reply:
Done